1.yaml

```diff
@@ -0,0 +1,38 @@
+package:
```

Comment:
I think we need to drop these 1 and 2

Reply:
good catch this is a dirty checkin ignore this